po/README.md

```diff
@@ -37,7 +37,7 @@ and commit your changes.
 - Languages can't be tested without installing Lutris via meson:
   ```bash
   rm -Rf transl-builddir
-  meson transl-builddir --prefix=~/.local
```

Comment:
Can you change the `rm -Rf` line as well?

Reply:
Or just every `transl-builddir` to `builddir` in the readme, if it's going to be changed once anyway.

Reply:
Is now ok?
Why lutris is not localized?
I try from terminal to call language:
env LANGUAGE=hr lutris
But all strings is on English.

Reply:
> Why lutris is not localized?

Because no-one changed all the strings inside the code. The only translatable strings for now are the ones in the ui files (which is not much). Also, Lutris doesn't default to Meson yet which is required for translations. If you want to test your translation, for this Readme: https://github.com/cvfosammmm/Setzer/blob/master/po/README.md (I haven't updated the one for Lutris yet)

> Is now ok?

Code changes yes, I guess you could squash 2e4e8a8a60f6c609cb497050446a40468fc9ae55 and 047638480fba848f62efe6b754e4daf29cd1f956 into one commit for better diffs.